homeassistant/components/overkiz/water_heater_entities/domestic_hot_water_production.py

```diff
@@ -158,30 +158,34 @@ def is_away_mode_on(self) -> bool | None:
     @property
     def min_temp(self) -> float:
         """Return the minimum temperature."""
-        min_temp = self.device.states[OverkizState.CORE_MINIMAL_TEMPERATURE_MANUAL_MODE]
+        min_temp = self.device.states.get(
+            OverkizState.CORE_MINIMAL_TEMPERATURE_MANUAL_MODE
```

Comment:
You can keep the previous code. See https://github.com/iMicknl/python-overkiz-api/blob/main/pyoverkiz/models.py#L400

We return None if the state is not found.

Reply:
I’ve posted a comment. You can rollback your modifications.

Reply:
Done.